cgi/product_multilingual.pl

```diff
@@ -1876,82 +1876,78 @@ ($$$$$$)
 HTML
 ;
 
-		if ($nid ne 'alcohol') {
+		if (($nid eq 'alcohol') or ($nid eq 'energy-kj') or ($nid eq 'energy-kcal')) {
+			my $unit = '';
+			
+			if (($nid eq 'alcohol')) { $unit = '% vol / °' } # alcohol in % vol / °
+			elsif (($nid eq 'energy-kj')) { $unit = 'kJ' }
+			elsif (($nid eq 'energy-kcal')) { $unit = 'kcal'}
```

Comment:
Does this work without semicolons terminating each line?

Reply:
>Every simple statement must be terminated with a semicolon, unless it is the final statement in a block, in which case the semicolon is optional. 

But for the sake of convention i will add them. :)

Reply:
Oh okay, that's news to me. 🙂 nvm then.